core/controllers/reader.py

```diff
@@ -919,8 +919,13 @@ def _record_completion_and_get_next_exp_id(
             current_node_id: str. The node ID of the chapter that the user just
                 completed.
 
+        Raises:
+            PageNotFoundException. If the ENABLE_NEW_STRUCTURE_VIEWER_UPDATES
```

Comment:
Drop "if". Just say "The ... flag is not set."

Reply:
Done